tests/cycpp_tests.py

```diff
@@ -585,16 +585,19 @@ def test_schemafilter():
         '    "<interleave>\\n"\n'
         '    "    <element name=\\"x\\">\\n"\n'
         '    "        <oneOrMore>\\n"\n'
-        '    "            <element name=\\"key\\">\\n"\n'
-        '    "                <data type=\\"int\\"/>\\n"\n'
-        '    "            </element>\\n"\n'
-        '    "            <element name=\\"val\\">\\n"\n'
-        '    "                <data type=\\"double\\"/>\\n"\n'
+        '    "            <element name=\\"item\\">\\n"\n'
```

Comment:
I think I appreciate the semantic value of this level of nesting....  it basically fills the role of stl::pair<T1,T2> right?  (Does that interpretation make sense?)  Similarly, I can imagine how this level of nesting greatly simplifies the semantics of interpreting the file.